### PR TITLE
release for v0.8.5

### PR DESCRIPTION
**Pull request by nabobalis**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -17,22 +17,20 @@ requirements:
   build:
     - python
     - setuptools
-    - numpy <1.14  # [py27]
```
Comment (Cadair) [inline recipe/meta.yaml]: :thinking: can you do `numpy != 1.14.0` lol?!

Comment (nabobalis) [reply recipe/meta.yaml]: Good question. Should I try?

Comment (Cadair) [reply recipe/meta.yaml]: :woman_shrugging:

Comment (nabobalis) [reply recipe/meta.yaml]: I did something but i have no idea what

Comment (Cadair) [reply recipe/meta.yaml]: it appears to have worked!

Comment (Cadair) [reply recipe/meta.yaml]: circle passed.

Comment (nabobalis) [reply recipe/meta.yaml]: doesnt that just mean the recipe has no issues with it?

Comment (Cadair) [reply recipe/meta.yaml]: it means it built on linux.

Comment (nabobalis) [reply recipe/meta.yaml]: Shrug

File: recipe/meta.yaml [hunk 2]
```diff
@@ -17,22 +17,22 @@ requirements:
...
-    - numpy >1.7.1  # [not py27]
-    - astropy >=2.0
+    - numpy >=1.11|!=1.14.0  # [py27]
+    - numpy >=1.11  # [not py27]
```
Comment (isuruf) [inline recipe/meta.yaml]: You should build with the oldest version of numpy that you want to support and keep the run requirements as it is here.

```
build:
  - numpy 1.11.*
run:
  - numpy >=1.11|!=1.14.0  # [py27]
  - numpy >=1.11  # [not py27]
```

Comment (nabobalis) [reply recipe/meta.yaml]: Yes that is a good point. I will update it for the next go around. Thanks!

**Pull request merged**